tests/end_to_end/http_server/multi_deploy/src/index.ts

```diff
@@ -1,42 +1,13 @@
-import { init, postUpgrade, Server } from 'azle/experimental';
 import express from 'express';
 
-let initCalled: boolean = false;
-let postUpgradeCalled: boolean = false;
+const app = express();
 
-export default Server(serverCallback, {
-    init: init([], () => {
-        console.info('Init was called');
-        initCalled = true;
-        postUpgradeCalled = false;
-    }),
-    postUpgrade: postUpgrade([], () => {
-        console.info('Post Upgrade was called');
-        postUpgradeCalled = true;
-        initCalled = false;
-    })
+app.get('/get-azle-init-called', (_req, res) => {
+    res.send((globalThis as any)._azleInitCalled);
```

Comment:
Let's try just doing import 'azle'; to get rid of the as any

Reply:
Looks good